tests/DocumentationExamplesTest.php

```diff
@@ -730,6 +731,71 @@ public function testExample_42_50(): void
         }
     }
 
+    public function testAggregationProjectionExample_1(): void
+    {
+        $this->skipIfServerVersion('<', '4.4.0', '$project syntax for find and findAndModify is not supported');
```

Comment:
Just realized that the projection syntax requires 4.4+, so I've broken this example out into a separate method and pulled in some data fixtures from the related test method.